cv32e40x/tests/programs/custom/generic_exception_test/generic_exception_test.S

```diff
@@ -208,7 +210,7 @@ main:
     csrrc x14, mstatus, x0
     nop
     nop
- 
+
 added_by_mike:
```

Comment:
Not part of this review per se, but maybe a more informative label would be useful here?

Reply:
Noted that general exception test cleanup should be done for E40X and future cores.